src/css/query_editor.css

```diff
@@ -21,3 +21,124 @@
 .vc-learn-more__link {
   text-decoration: underline;
 }
+
+.vc-query-editor__loader-container {
```

Comment:
Taken from https://projects.lukehaas.me/css-loaders/. Should I add a comment?

Reply:
Maybe a "thanks" area in our readme? It's MIT so we are good to go.